supertokens_python/syncio/__init__.py

```diff
@@ -61,12 +61,12 @@ def get_user_count(
     tenant_id: Optional[str] = None,
 ) -> int:
     return sync(
-        Supertokens.get_instance().get_user_count(include_recipe_ids, tenant_id)
+        Supertokens.get_instance().get_user_count(include_recipe_ids, None, tenant_id)
```

Comment:
see comment in asyncio file. User_context should be passed in from the user (Default to None), and that needs to be propogated